engine/gui/src/gui.cpp

```diff
@@ -2923,6 +2926,10 @@ namespace dmGui
     void SetNodeResetPoint(HScene scene, HNode node)
     {
         InternalNode* n = GetNode(scene, node);
+        if (!n->m_Node.m_ResetPointProperties)
+        {
+            n->m_Node.m_ResetPointProperties = (dmVMath::Vector4*)malloc(sizeof(dmVMath::Vector4) * PROPERTY_COUNT);
+        }
```

Comment:
`m_Node.m_ResetPointProperties` is rarely needed when we reset values, so we don't need these values all the time as part of the node structure. It helps to reduce node structure size to 512 bytes instead of 720 which should improve cache locality